server/modules/auth/auth.service.ts

```
@@ -1,16 +1,16 @@
-import { Types } from 'mongoose';
-import { sendEmail } from '../../utils/email';
-import ErrorResponse from '../../utils/errorResponse';
-import Token from '../token/token.model';
-import { PermissionScope } from './../permission/permission.model';
-import User, { UserDocument } from './../user/user.model';
-
-import UserRoleAssignment from '../role_assignment/userRoleAssignment.model';
+import { organizationService } from "./../organization/organization.service";
+import { ObjectId } from "mongoose";
+import bcrypt from "bcryptjs";
+import crypto from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)